<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Petr Vanek &lt;petr@scribus.info&gt;</author>
 <comment>Use designer to update source ui file. All manual changes in cpp/h files will be lost</comment>
 <class>GuideManager</class>
 <widget class="ScrPaletteBase" name="GuideManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>432</width>
    <height>389</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Guide Manager</string>
  </property>
  <layout class="QGridLayout">
   <property name="leftMargin">
    <number>11</number>
   </property>
   <property name="topMargin">
    <number>11</number>
   </property>
   <property name="rightMargin">
    <number>11</number>
   </property>
   <property name="bottomMargin">
    <number>11</number>
   </property>
   <property name="spacing">
    <number>6</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="standardPage">
      <attribute name="title">
       <string>&amp;Single</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0" colspan="2">
        <layout class="QHBoxLayout">
         <item>
          <widget class="QGroupBox" name="horizontalGroupBox">
           <property name="title">
            <string>Horizontals</string>
           </property>
           <layout class="QGridLayout">
            <item row="0" column="0" colspan="2">
             <widget class="GuidesView" name="horizontalView">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="addHorButton">
              <property name="toolTip">
               <string>Add a new horizontal guide</string>
              </property>
              <property name="text">
               <string>&amp;Add</string>
              </property>
              <property name="shortcut">
               <string>Alt+A</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="delHorButton">
              <property name="toolTip">
               <string>Delete the selected horizontal guide</string>
              </property>
              <property name="text">
               <string>D&amp;elete</string>
              </property>
              <property name="shortcut">
               <string>Alt+E</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="verticalGroupBox">
           <property name="title">
            <string>Verticals</string>
           </property>
           <layout class="QGridLayout">
            <item row="0" column="0" colspan="2">
             <widget class="GuidesView" name="verticalView">
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="addVerButton">
              <property name="toolTip">
               <string>Add a new vertical guide</string>
              </property>
              <property name="text">
               <string>A&amp;dd</string>
              </property>
              <property name="shortcut">
               <string>Alt+D</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QPushButton" name="delVerButton">
              <property name="toolTip">
               <string>Delete the selected vertical guide</string>
              </property>
              <property name="text">
               <string>De&amp;lete</string>
              </property>
              <property name="shortcut">
               <string>Alt+L</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QCheckBox" name="lockCheck">
         <property name="toolTip">
          <string>Lock the guides</string>
         </property>
         <property name="text">
          <string>&amp;Lock Guides</string>
         </property>
         <property name="shortcut">
          <string>Alt+L</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QPushButton" name="applyToAllStdButton">
         <property name="toolTip">
          <string>Apply to all pages</string>
         </property>
         <property name="text">
          <string>Appl&amp;y to All Pages</string>
         </property>
         <property name="shortcut">
          <string>Alt+Y</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="autoPage">
      <attribute name="title">
       <string>&amp;Column/Row</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QGroupBox" name="horizontalAutoGroup">
         <property name="title">
          <string>Horizontals</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel1">
            <property name="text">
             <string>&amp;Number:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>horizontalAutoCountSpin</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ScrSpinBox" name="horizontalAutoCountSpin">
            <property name="toolTip">
             <string>Number of horizontal guides to create</string>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="horizontalAutoGapCheck">
            <property name="toolTip">
             <string>Create rows with guides, with an additional gap between the rows</string>
            </property>
            <property name="text">
             <string>U&amp;se Gap:</string>
            </property>
            <property name="shortcut">
             <string>Alt+S</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="ScrSpinBox" name="horizontalAutoGapSpin"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>Refer to</string>
            </property>
            <layout class="QGridLayout">
             <item row="0" column="0">
              <widget class="QRadioButton" name="horizontalPageAutoButton">
               <property name="toolTip">
                <string>Create the selected number of horizontal guides relative to the current page</string>
               </property>
               <property name="text">
                <string>&amp;Page</string>
               </property>
               <property name="shortcut">
                <string>Alt+P</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="horizontalMarginsAutoButton">
               <property name="toolTip">
                <string>Create the selected number of horizontal guides relative to the current page's margins</string>
               </property>
               <property name="text">
                <string>M&amp;argins</string>
               </property>
               <property name="shortcut">
                <string>Alt+A</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="horizontalSelectionAutoButton">
               <property name="toolTip">
                <string>Create the selected number of horizontal guides relative to the current selection of items</string>
               </property>
               <property name="text">
                <string>S&amp;election</string>
               </property>
               <property name="shortcut">
                <string>Alt+E</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="verticalAutoGroup">
         <property name="title">
          <string>Verticals</string>
         </property>
         <layout class="QGridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="textLabel1_2">
            <property name="text">
             <string>Nu&amp;mber:</string>
            </property>
            <property name="wordWrap">
             <bool>false</bool>
            </property>
            <property name="buddy">
             <cstring>verticalAutoCountSpin</cstring>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="ScrSpinBox" name="verticalAutoCountSpin">
            <property name="toolTip">
             <string>Number of vertical guides to create</string>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="minimum">
             <number>0</number>
            </property>
            <property name="value">
             <number>0</number>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="QCheckBox" name="verticalAutoGapCheck">
            <property name="toolTip">
             <string>Create columns with guides, with an additional gap between the columns</string>
            </property>
            <property name="text">
             <string>Use &amp;Gap:</string>
            </property>
            <property name="shortcut">
             <string>Alt+G</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0" colspan="2">
           <widget class="ScrSpinBox" name="verticalAutoGapSpin"/>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QGroupBox" name="groupBox_2">
            <property name="title">
             <string>Refer to</string>
            </property>
            <layout class="QGridLayout">
             <item row="0" column="0">
              <widget class="QRadioButton" name="verticalPageAutoButton">
               <property name="toolTip">
                <string>Create the selected number of vertical guides relative to the current page</string>
               </property>
               <property name="text">
                <string>&amp;Page</string>
               </property>
               <property name="shortcut">
                <string>Alt+P</string>
               </property>
               <property name="checked">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QRadioButton" name="verticalMarginsAutoButton">
               <property name="toolTip">
                <string>Create the selected number of vertical guides relative to the current page's margins</string>
               </property>
               <property name="text">
                <string>M&amp;argins</string>
               </property>
               <property name="shortcut">
                <string>Alt+A</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0">
              <widget class="QRadioButton" name="verticalSelectionAutoButton">
               <property name="toolTip">
                <string>Create the selected number of vertical guides relative to the current selection of items</string>
               </property>
               <property name="text">
                <string>S&amp;election</string>
               </property>
               <property name="shortcut">
                <string>Alt+E</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0" colspan="2">
        <widget class="QPushButton" name="applyToAllAutoButton">
         <property name="toolTip">
          <string>Apply the shown guides to all pages in the document</string>
         </property>
         <property name="text">
          <string>Appl&amp;y to All Pages</string>
         </property>
         <property name="shortcut">
          <string>Alt+Y</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="miscPage">
      <attribute name="title">
       <string>&amp;Misc</string>
      </attribute>
      <layout class="QGridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="deletePageButton">
         <property name="toolTip">
          <string>Delete all guides from the current page</string>
         </property>
         <property name="text">
          <string>Delete Guides from Current &amp;Page</string>
         </property>
         <property name="shortcut">
          <string>Alt+P</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="deleteAllGuides">
         <property name="toolTip">
          <string>Delete all guides from the current document</string>
         </property>
         <property name="text">
          <string>Delete Guides from &amp;All Pages</string>
         </property>
         <property name="shortcut">
          <string>Alt+A</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <spacer>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Expanding</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">ui/scrspinbox.h</header>
  </customwidget>
  <customwidget>
   <class>ScrPaletteBase</class>
   <extends>QWidget</extends>
   <header>ui/scrpalettebase.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>GuidesView</class>
   <extends>QTableView</extends>
   <header>guidesview.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local">ui/scrpalettebase.h</include>
  <include location="local">ui/scrspinbox.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
